src/pages/VolunteerDashboard.jsx

```diff
@@ -1,22 +1,55 @@
 import '../styles/AdminDashboard.css';
 import { Row, Col } from 'antd';
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 import UpcomingEvents from '../components/admin-dashboard-components/UpcomingEvents';
 import DashboardHeader from '../components/admin-dashboard-components/DashboardHeader';
 import PastEvents from '../components/admin-dashboard-components/PastEvents';
+import EventList from '../components/events/EventList';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module '../components/events/EventList'.